.github/actions/lint-front-matter/rules/url-matches-title.js

```diff
@@ -0,0 +1,29 @@
+const id = 'url-matches-title';
+const Rule = require('./rule');
+const path = require('path');
+const slugify = require('slugify');
+const {ok, fail} = require('../utils/status');
+
+/**
+ * @param {string} file Filename of the file being linted.
+ * @param {Object} yaml The yaml front matter from the file.
+ * @return {Object} The result of the check and any failure/warning messages.
+ */
+const test = (file, yaml) => {
+  const dirname = path
+    .dirname(file)
+    .split('/')
+    .pop();
+  const sluggifiedTitle = slugify(yaml.title, {lower: true, strict: true});
```

Comment:
This will fail on a bunch of existing posts I think. Not a huge deal, just a note.

Reply:
yeah i think this is only being run against newly added posts in added-files. And we may decide to scrap this rule entirely.

Reply:
@kaycebasques @jpmedley do y'all want this rule? It basically fails if the url doesn't match the sluggified title. A reviewer can add a label to skip this test if they approve of the title and url. Another option would be to warn, or not bother flagging it at all.

Reply:
I don't think this is useful. It will result in either very long slugs or very short titles. I don't think either serves our readers.